run_update.py

```diff
@@ -347,6 +347,9 @@ def get_projects(organization):
                         # some values might be empty strings
                         elif type(project_value) in (str, unicode) and unicode(project_value.decode('utf8')) == u'':
                             project[project_key] = None
+                        # we want tags to be a list with no whitespace
+                        elif project_key == 'tags':
+                            project[project_key] = [tag.strip() for tag in project_value.split(',')]
```

Comment:
Won't lists get caught by the first condition? (`if type(project_value) is list`)

If it won't, we should encode the tags before we save them, maybe add `project_value = unicode(project_value.decode('utf8'))` before splitting it into the list.